src/main/java/cubicchunks/util/FastCubeBlockAccess.java

```diff
@@ -168,11 +151,13 @@ private IBlockState getBlockState(int blockX, int blockY, int blockZ) {
         return Blocks.AIR.getDefaultState();
     }
 
-    @Override public int getBlockLightOpacity(BlockPos pos) {
+    @Override 
+    public int getBlockLightOpacity(BlockPos pos) {
```

Comment:
Does the formatter actually do that?

Reply:
Yes, CubicChunk Eclipse formatter do that if I launch it on those functions.